blockchain/src/blockchain/abstract_blockchain.rs

```diff
@@ -43,6 +46,20 @@ impl AbstractBlockchain for Blockchain {
         self.state.main_chain.head.epoch_number()
     }
 
+    fn can_enforce_validity_window(&self) -> bool {
```

Comment:
Would it you be able to remove the method from the blockchain wrapper(blockchain/src/blockchain/wrappers.rs:238)?

If you can enforce the validity window is now defined in two places.

Reply:
Yup, good catch!
I intended to remove this from the blockchain wrapper.